containers/themes/ThemesSection.js

```diff
@@ -0,0 +1,108 @@
+// const CSSBlueTheme =
+//     '/* blue-theme */.aside {\r\n  background: #526ee0; }\r\n\r\n.logo g {\r\n  fill: #f6f7fa; }\r\n\r\n.toolbar {\r\n  background: #788ee8; }\r\n\r\n.content {\r\n  background-image: -webkit-linear-gradient(top, #526ee0, #788ee8);\r\n  background-image: linear-gradient(to bottom, #526ee0, #788ee8); }\r\n\r\n.navigation__link,\r\n.navigation__link:focus,\r\n.navigation__link:hover,\r\n.animated-expandmore__button {\r\n  color: #e3e4e7; }\r\n\r\n[aria-expanded="true"].animated-expandmore__button,\r\n[aria-current="page"].navigation__link,\r\n.animated-expandmore__button:active {\r\n  background: #526ee0;\r\n  color: #fff; }\r\n\r\n.has-scroll-inside-navigation > .spacebar-container::before {\r\n  background: linear-gradient(to top, #526ee0 0, rgba(0, 0, 0, 0) 2em); }\r\n\r\n.aside-link {\r\n  background: rgba(0, 0, 0, .3); }\r\n\r\n.searchbox-field[type="search"] {\r\n  background-color: rgba(0, 0, 0, .2);\r\n  border-color: #526ee0; }\r\n\r\n.spacebar {\r\n  background: rgba(255, 255, 255, .1); }\r\n\r\n.spacebar::-webkit-meter-bar {\r\n  background: rgba(255, 255, 255, .1); }\r\n\r\n.topnav-link {\r\n  color: #fff; }\r\n  .topnav-link:focus, .topnav-link:hover, .topnav-link:active, .topnav-link[aria-current="true"] {\r\n    color: #fff; }\r\n    .topnav-link:focus > .topnav-icon, .topnav-link:hover > .topnav-icon, .topnav-link:active > .topnav-icon, .topnav-link[aria-current="true"] > .topnav-icon {\r\n      fill: #fff; }\r\n  .topnav-link[aria-current="true"] {\r\n    border-color: #fff; }\r\n\r\n.searchbox-advanced-search-button svg,\r\n.topnav-link svg {\r\n  fill: #fff; }\r\n\r\n.main {\r\n  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.1); }';
+
+const CSSLightTheme = '/* light-theme */';
```

Comment:
Extract these constants in `proton-shared/lib/constants` please.

Reply:
These are just temporary. I'm waiting for Nico to provide with the CSS. These constants will never be in the code, actually. The CSS will be generated from a function in `design-system`.